cpp/include/cudf/functions.h

```diff
@@ -393,53 +393,7 @@ gdf_error gdf_hash_partition(int num_input_cols,
  * returned.
  */
 /* ----------------------------------------------------------------------------*/
-gdf_error gdf_prefixsum_generic(gdf_column *inp, gdf_column *out, int inclusive);
-
-/* --------------------------------------------------------------------------*/
-/** 
- * @Synopsis  Computes the prefix sum of a column
- * 
- * @Param inp Input column for prefix sum with null_count = 0
- * @Param out The output column containing the prefix sum of the input
- * @Param inclusive Flag for applying an inclusive prefix sum
- * 
- * @Returns   GDF_SUCCESS if the operation was successful, otherwise an appropriate
- * error code. If inp->null_count is not set to 0 GDF_VALIDITY_UNSUPPORTED is
- * returned.
- */
-/* ----------------------------------------------------------------------------*/
-gdf_error gdf_prefixsum_i8(gdf_column *inp, gdf_column *out, int inclusive);
-
-/* --------------------------------------------------------------------------*/
-/** 
- * @Synopsis  Computes the prefix sum of a column
- * 
- * @Param inp Input column for prefix sum with null_count = 0
- * @Param out The output column containing the prefix sum of the input
- * @Param inclusive Flag for applying an inclusive prefix sum
- * 
- * @Returns   GDF_SUCCESS if the operation was successful, otherwise an appropriate
- * error code. If inp->null_count is not set to 0 GDF_VALIDITY_UNSUPPORTED is
- * returned.
- */
-/* ----------------------------------------------------------------------------*/
-gdf_error gdf_prefixsum_i32(gdf_column *inp, gdf_column *out, int inclusive);
-
-/* --------------------------------------------------------------------------*/
-/** 
- * @Synopsis  Computes the prefix sum of a column
- * 
- * @Param inp Input column for prefix sum with null_count = 0
- * @Param out The output column containing the prefix sum of the input
- * @Param inclusive Flag for applying an inclusive prefix sum
- * 
- * @Returns   GDF_SUCCESS if the operation was successful, otherwise an appropriate
- * error code. If inp->null_count is not set to 0 GDF_VALIDITY_UNSUPPORTED is
- * returned.
- */
-/* ----------------------------------------------------------------------------*/
-gdf_error gdf_prefixsum_i64(gdf_column *inp, gdf_column *out, int inclusive);
-
+gdf_error gdf_prefixsum(gdf_column *inp, gdf_column *out, int inclusive);
```

Comment:
Can you add additional detail to the Doxymentation that describes the behavior for `inclusive`? I.e., does 0 mean exclusive and non-zero means inclusive? Or does it need to be `1`?